test/server_test.coffee

```diff
@@ -102,3 +103,12 @@ describe 'a mock API server', ->
       assert.equal JSON.parse(body).answer, "Not Found"
       assert.equal 404, statusCode
       done()
+
+  it 'supports configurable test-path', ->
+    options =
+      testPath: 'test/mock-api-custom-path2'
+
+    doRequest options, identity, ({body, statusCode}) ->
+      assert.equal JSON.parse(body).answer, "Hello, World!"
```

Comment:
This would succeed if `--test-path` doesn't work, since the default test path also has a `hello.json` with the same contents.  I would recommend changing the name of the file or its contents.